src/test/java/org/mule/extensions/jms/test/internal/operation/profiling/tracing/JmsSpanCustomizerTestCase.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.extensions.jms.test.internal.operation.profiling.tracing;
+
+import static org.mule.extensions.jms.api.destination.DestinationType.TOPIC;
+import static org.mule.extensions.jms.internal.operation.profiling.tracing.JmsConsumeSpanCustomizer.getJmsConsumeSpanCustomizer;
+import static org.mule.extensions.jms.internal.operation.profiling.tracing.JmsPublishSpanCustomizer.getJmsPublishSpanCustomizer;
+import static org.mule.extensions.jms.test.AllureConstants.JmsFeature.JmsStory.TRACING;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.mule.extensions.jms.api.destination.DestinationType;
+import org.mule.extensions.jms.api.message.JmsMessageBuilder;
+import org.mule.extensions.jms.internal.operation.profiling.tracing.JmsConsumeSpanCustomizer;
+import org.mule.extensions.jms.internal.operation.profiling.tracing.JmsPublishSpanCustomizer;
+import org.mule.jms.commons.api.destination.ConsumerType;
+import org.mule.jms.commons.internal.connection.JmsTransactionalConnection;
+import org.mule.runtime.api.metadata.DataType;
+import org.mule.runtime.api.metadata.TypedValue;
+import org.mule.sdk.api.runtime.source.DistributedTraceContextManager;
+
+import java.util.Locale;
+import java.util.OptionalLong;
+
+import javax.jms.Connection;
+import javax.jms.ConnectionMetaData;
+import javax.jms.JMSException;
+
+import io.qameta.allure.Description;
+import io.qameta.allure.Issue;
+import io.qameta.allure.Story;
+import org.junit.Test;
+
+@Story(TRACING)
+@Issue("W-11859222")
```

Comment:
I didn't know this annotation. Please, could you give me more information about the "issue annotation"?

Reply:
Hi! We usually use this annotation to link the issue in which a new feature was added / a bug was solved with the corresponding tests